docs/src/blockchain-development/calling_contracts.md

```diff
@@ -0,0 +1,127 @@
+# Calling Contracts
+
+End users and smart contracts typically need to communicate with each other.
+
+The FuelVM achieves end user and inter contract communication primarily with the [`call`](https://github.com/FuelLabs/fuel-specs/blob/master/specs/vm/opcodes.md#call-call-contract) opcode.
+
+Sway provides a nice way manage calls with its `abi` system.
+
+## Script Calls
+
+Here is an example of a *script calling a contract* in Sway:
+
+```rs
```

Comment:
use `sway` instead of `rs` for Sway snippets.

Reply:
Done.